javascript-view-swapping/main.js

```diff
@@ -0,0 +1,35 @@
+var tabContainer = document.querySelector('.tab-container');
+var tab = document.querySelectorAll('.tab');
+var view = document.querySelectorAll('.view');
+
+tabContainer.addEventListener('click', tabChange);
+
+function tabChange(event) {
+
+  if (event.target.matches('.tab') !== true) {
+    return;
+  }
+
+  for (var i = 0; i < tab.length; i++) {
+    if (tab[i] === event.target) {
+      tab[i].className = 'tab active';
+    } else {
+      tab[i].className = 'tab';
+    }
+  }
+
+  var dataView = event.target.getAttribute('data-view');
+
+  if (event.target.matches('.tab') !== true) {
+    return;
+  }
```

Comment:
You already did this conditional above, you don't need to check the same thing twice in one function.